packages/beacon-node/src/chain/blocks/importBlock.ts

```diff
@@ -212,6 +213,28 @@ export async function importBlock(
   const newHead = this.recomputeForkChoiceHead(ForkchoiceCaller.importBlock);
   const currFinalizedEpoch = this.forkChoice.getFinalizedCheckpoint().epoch;
 
+  // Suppress fcu call if this is true
+  let shouldOverrideFcu = false;
+
+  if (opts.isGossipBlock && isExecutionStateType(postState)) {
+    const proposerIndex = postState.epochCtx.getBeaconProposer(blockSlot + 1);
```

Comment:
this `getBeaconProposer` can throw so either need to do a try/catch here
if `blockSlot` is end of this epoch we can't get proposer of next slot so maybe handle it separately

Reply:
If `blockSlot` is end of epoch, we don't reorg anyway due to this condition
https://github.com/ChainSafe/lodestar/blob/3c7d6d16870d79a8eb69b718930c171d11587f14/packages/fork-choice/src/forkChoice/forkChoice.ts#L1480

And we don't record `NotReorgedReason` here, so there is no need to handle this separately.